index.src.html

```diff
@@ -728,29 +733,29 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
   The <dfn method for=ReportingObserver><code>observe()</code></dfn>
   method, when invoked, must run these steps:
 
-  1. Let |environment| be the <a spec=ecmascript lt=realm>ECMAScript global
-     environment</a> associated with the <a>context object</a>.
+  1. Let |global| be the <a spec=ecmascript>global object</a> of the <a
+     spec=ecmascript>realm</a> associated with the <a>context object</a>.
```

Comment:
This can be `Let |global| be the [=relevant global object=] of [=this=]`. (Although perhaps you should keep using "context object" for consistency for now, instead of "this".)

Reply:
Done. (Will switch to [=this=] soon)